### PR TITLE
Visible message notice-spanclasses to no spanclass

### PR DESCRIPTION
**Pull request by kingofkosmos**
## About The Pull Request
No spanclass (just plain text) is usually used for messages which tell the player they're seeing something unalarming.
This changes all the current visible 'notice'-spans to no spanclass. Also changes some spanclasses to be more appropriate to the situation.

## Why It's Good For The Game
Makes spanclasses more consistent, helping players to better spot important things from the chat.

### CODE REVIEW

**Comment by AnturK** [comment]
Bad, if you want to change styling change the class don't remove the spans.

**Comment by kingofkosmos** [comment]
Do we have a span class which is looks like plain text?

**Comment by AnturK** [comment]
Not sure, but if we don't just add one.

**Comment by cacogen** [comment]
I have really bad eyesight and these help me tell things apart in the chat. I was glad when they were added. Please don't remove them.

**Comment by kingofkosmos** [comment]
> I have really bad eyesight and these help me tell things apart in the chat, which I find it easy to miss things in. I was glad when they were added. Please don't remove them.

Tell which things apart? I'm trying exactly help people tell things apart by making the spans more organized.

**Comment by Akrilla** [comment]
The point of notice spans is to style actions that "have something happening". Unstyling things to literally blend into text is a clear net loss to overall readability. People can very easily differentiate between the danger, warnings etc, there's no need to remove notice spans.

**Comment by Akrilla** [comment]
Any text that currently is unstyled but printed to the user I'd say is a mistake and instead *should* be notice span'd.

**Comment by kingofkosmos** [comment]
>Any text that currently is unstyled but printed to the user I'd say is a mistake and instead should be notice span'd.

That would mean that the most of the current visible_messages are mistakenly not spanned. There's 634 cases of no-span, and 473 cases of notice-span in the first message of all visible_messages.

>The point of notice spans is to style actions that "have something happening". 

What I'm proposing here is that we separate "have something happening separate from you" and "have something happen which you're doing". This would improve overall readability.

**Comment by cacogen** [comment]
> What I'm proposing here is that we separate "have something happening
separate from you" and "have something happen which you're doing". This
would improve overall readability.

I agree with this actually. Warning and danger messages from other people
will still be red. I'm just worried the black messages will be too easy to
overlook. But then I'm wondering what other black messages there are.

**Comment by kingofkosmos** [comment]
Alright all the 'notice' spans have been changed to 'none'-spanclasses. Since there's no style set to 'none', they will default to plain text. Maybe I'll have another PR where I add 'none'-spans to the 634 other visible_message texts missing spans.

And sorry about the messy commits, (didn't know how to undo a commit properly) should I do something about them?

**Comment by Akrilla** [comment]
> That would mean that the most of the current visible_messages are mistakenly not spanned. There's 634 cases of no-span, and 473 cases of notice-span in the first message of all visible_messages.

Correct, I'd say they should be notice'd. Though to be fair I don't know what these are saying - I've never come across a visible message that wasn't spanned in some way.

> What I'm proposing here is that we separate "have something happening separate from you" and "have something happen which you're doing". This would improve overall readability.

Fundamentally I think anything that blends into the same colour of regular spoken conversation and /me's is making readability worse.

File: code/datums/components/butchering.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@
 			new sinew (T)
 		meat.guaranteed_butcher_results.Remove(sinew)
 	if(butcher)
-		meat.visible_message("<span class='notice'>[butcher] butchers [meat].</span>")
+		meat.visible_message("<span class='none'>[butcher] butchers [meat].</span>")
```
Comment (Akrilla) [inline code/datums/components/butchering.dm]: If you're keeping unstyled classes, the class name probably shouldn't be none. In CSS this tends to mean it doesn't show.

Comment (kingofkosmos) [reply code/datums/components/butchering.dm]: You mean specifically `span class='none'`? Could it be `span class='plain'` or something similar?

**Comment by kingofkosmos** [comment]
>Fundamentally I think anything that blends into the same colour of regular spoken conversation and /me's is making readability worse.

This is a very good point and I'd like to hear what the admins have to say. I'd be happy to change them all to 'notice' if it improves readability.

File: code/datums/components/butchering.dm (hunk 1 above)
Comment (Akrilla) [reply code/datums/components/butchering.dm]: Yeah plain is fine, just the word none tends to imply a class that has thever "display: none" property, which hides elements.

**Comment by kingofkosmos** [comment]
Pinging @Cyberboss and @ninjanomnom, what do you think, should all of these be made:
'notice', or
'plain' (default black text), or
some new 'fluff' which would be a lighter blue font or something like that?

Whatever is the decision here, I'll make another PR and convert all the rest of the visible_messages to it as well.

**Comment by ExcessiveUseOfCobblestone** [comment]
talking/emoting should be a distinct format from actions

**Comment by ninjanomnom** [comment]
Yes, ping the absent maintainer and the maintainer which couldn't care less about UI.

At the very least though the span class name should be named after what it's for not what it is.

**Comment by kingofkosmos** [comment]
Alright thanks, I'll make a new PR for changing all mundane actions to be set to 'notice'.

**Pull request closed without merging**